packages/web/next-seo.config.tsx

```diff
@@ -0,0 +1,74 @@
+import { DefaultSeo, DefaultSeoProps } from "next-seo";
+import { useTranslation } from "react-multi-lang";
+
+import { IS_FRONTIER } from "~/config/index";
+import spriteSVGURL from "~/public/icons/sprite.svg";
+
+const SEO: React.FC = () => {
+  const t = useTranslation();
+
+  const SEO_VALUES = {
+    SITE_URL: "https://osmosis.zone/",
+    SITE_TITLE: IS_FRONTIER
+      ? t("seo.default.titleFrontier")
+      : t("seo.default.title"),
+    SITE_DESCRIPTION: t("seo.default.description"),
+    TWITTER_HANDLE: "@osmosiszone",
+    IMAGE_PREVIEW: "/images/preview.jpg",
+    FAVICON: "/favicon.ico",
+    SHORTCUT_ICON: `${
+      typeof window !== "undefined" ? window.origin : ""
```

Comment:
I'm assuming this is because the window is undefined on first render (due to next server rendering this). Will the diff server/client HTMLs affect SEO? I honestly don't know. It's probably better to move it to a `useEffect` hook so that we avoid any potential issue.

```
  const [shortcutIcon, setShortcutIcon] = useState<string>('');

  useEffect(() => {
    setShortcutIcon(`${window.origin}/osmosis-logo-wc.png`);
  }, []);
  ```